assets/queries/cloudFormation/memcached_disabled/test/negative2.json

```diff
@@ -0,0 +1,22 @@
+{
+  "Description": "A sample template",
+  "Resources": {
+    "ElasticacheCluster": {
```

Comment:
Change "ElasticacheCluster" to "ElasticacheCluster2"

Reply:
Why does it need to be change from sample to sample?

Reply:
I think all the samples of a query are grouped together, at some point, and having the same name can lead to conflicts. At least, I have seen PRs that the only thing they changed was differentiating the resource names, so I thought better safe than sorry.

Reply:
Is this part of the engine? End-users can't submit different files with the same resources' names without the risk of conflicts?
@ruigomescx if you could have an input on this as well

Reply:
done